frontend/src/modules/controllers/AlarmModal.tsx

```diff
@@ -110,30 +105,19 @@ interface Props {
  *
  * @returns JSX.Element
  */
-export const AlarmModal = ({
+export const AlarmModalContent = ({
```

Comment:
The jsdoc code comment for this `AlarmModalContent` component should be updated, and a code comment should be added for the new `AlarmModal` component; together, the two jsdoc code comments should explain what is the division of responsibilities between the two components.